en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+An organization exposes a flight tracking API as a managed API on Choreo. This system involves several APIs and databases. The scenario and the APIs in the system are described below:
+
+#### Scenario
+
+To search for real-time flight data, a client will enter search criteria, and search through the web application. Upon receiving the request, the client application (web application) will make a call to the [Flights Status API](#flightStatusAPI), a managed API on Choreo, to search for flight data. Choreo will then invoke the backend API, the [Real-Time Flight API](#realtimeFlightAPI). The backend API will invoke a database to get the information and pass it to the client.
+
+#### Glossary
+
+- <a class="pricing" id="flighStatusAPI">Flight Status API</a>: An external-facing, managed API on Choreo. This API exposes the Real-Time Flight API on Choreo as a managed API.
+
+- <a class="pricing" id="realtimeFlightAPI">Real-TIme Flight API</a>: An unmanaged, external-facing API hosted outside of Choreo.
+
+- <a class="pricing" id="flightsDatabase">Flights Database</a>: A database that stores flight information.
+
+
+#### Get flight status
+
+![Get flight status](../assets/img/references/api-proxy-example1.png)
+
+Let’s see how the steps are calculated for the get flight status process:
+
+- As illustrated above, the client initiates a search request for flight data through the web application. 
+- The web application then makes **1 API call to the [Flight Status API](#flightStatusAPI)** in Choreo. This is counted as **1 step**.
+- The Flight Status API in Choreo then makes **1 call to the [Real-Time Flight API](#realtimeFlightAPI)** with the search criteria entered by the client. This is counted as **1 step**. 
+- The Real-Time Flight API then invokes the database to retrieve the information.
+
+- **Assuming the entire execution time for the get flight status process takes less than 500ms** (starting from making a call to Choreo, and Choreo making a call to the backend API),the get flight status process thereby results in a **total of 2 steps**.
+
+<br/>
+### Example 3: Webhook
+An airport has its flight information display system (FIDS) in Choreo.  This system involves APIs and webhooks.
+
+#### Scenario
+This scenario describes a hypothetical maintenance process carried out by airline maintenance. The airline uses Choreo to generate alerts by using Webhooks when the aircraft maintenance status is updated. 
+The Aircraft Maintenance team carries out maintenance for each flight before take-off. A supervisor updates the maintenance status of the aircraft on a [Aircraft Maintenance Sheet](#AircraftMaintenanceSheet), which is a Google sheet. Upon updating the sheet, the [Aircraft Status Notifier](#aircraftNotifWebhook) webhook on Choreo gets triggered and updates the [Flight Scheduling Database](#flightSchedulingDatabase) that confirms if the aircraft is ready for take off,  if it will be delayed due to maintenance, or if it is not suitable for take off.
```

Comment:
In the last part of the sentence, "whether" is more appropriate than "if".

"...that confirms whether the aircraft is ready to take off, whether it will be delayed due to maintenance, or whether it is not suitable for take off."

Reply:
+1. Changed